webview/src/shared/components/contextMenu/ContextMenu.tsx

```diff
@@ -56,7 +56,7 @@ export const ContextMenu: React.FC<ContextMenuProps> = ({
   <Tooltip
     trigger={trigger}
     delay={[100, 200]}
-    placement={'bottom'}
+    placement={'right-start'}
```

Comment:
All possible placements are [here](https://atomiks.github.io/tippyjs/#placements).

Reply:
should we do right-end?

Reply:
You would think so but:


https://user-images.githubusercontent.com/37993418/189559868-9a5be8ec-d7f0-419e-9e18-cb93f2f394e5.mov

Reply:
Normally it's bottom-end. No?

Reply:
bottom-end:


https://user-images.githubusercontent.com/37993418/189771989-477e192d-e26c-4425-b143-8cea497ce1bd.mov

Reply:
Maybe I was not clear enough, sorry. This is an example from Figma 

<img width="895" alt="Screen Shot 2022-09-12 at 7 26 32 PM" src="https://user-images.githubusercontent.com/98249521/189775519-83687542-da66-4f1a-bc25-42abb62e3b66.png">